Composite.Web.Css.Sass/Composite.Web.Css.Sass/CssCompilationHttpModule.cs

```diff
@@ -26,6 +27,11 @@ public CssCompilationHttpModule(string extension, string fileWatcherMask, Action
 
         public void Init(HttpApplication application)
         {
+            bool packageEnabled = ConfigurationManager.AppSettings["Orckestra.Web.Css.Sass"] == "true";

```

Comment:
Since there could be another settings related to the package, the setting should be called:
"Orckestra.Web.Css.Sass.Enabled"